pkg/diagnostics/service_monitoring.go

```diff
@@ -12,16 +14,20 @@ import (
 
 // Tag keys.
 var (
-	componentKey      = tag.MustNewKey("component")
-	failReasonKey     = tag.MustNewKey("reason")
-	operationKey      = tag.MustNewKey("operation")
-	actorTypeKey      = tag.MustNewKey("actor_type")
-	trustDomainKey    = tag.MustNewKey("trustDomain")
-	namespaceKey      = tag.MustNewKey("namespace")
-	policyActionKey   = tag.MustNewKey("policyAction")
-	resiliencyNameKey = tag.MustNewKey("name")
-	policyKey         = tag.MustNewKey("policy")
-	componentNameKey  = tag.MustNewKey("componentName")
+	componentKey        = tag.MustNewKey("component")
+	failReasonKey       = tag.MustNewKey("reason")
+	operationKey        = tag.MustNewKey("operation")
+	actorTypeKey        = tag.MustNewKey("actor_type")
+	trustDomainKey      = tag.MustNewKey("trustDomain")
+	namespaceKey        = tag.MustNewKey("namespace")
+	policyActionKey     = tag.MustNewKey("policyAction")
+	resiliencyNameKey   = tag.MustNewKey("name")
+	policyKey           = tag.MustNewKey("policy")
+	componentNameKey    = tag.MustNewKey("componentName")
+	destinationAppIDKey = tag.MustNewKey("dst_app_id")
```

Comment:
Can this be simplified to "dst_app"

Reply:
Lets keep this `dst_app_id` as appID is a known term across Dapr.